docs/cromwell_features/HogFactors.md

```diff
@@ -0,0 +1,163 @@
+# Hog Factors
+
+As of Cromwell 35, Cromwell introduces the concept hog factor (along with the related of hog groups and hog limits).
+
+## Concepts
+
+### Hog Group
+
+- Every workflow is assigned to a hog group when Cromwell receives it. Exactly how this happens is [configurable](#configuration).
+- Any sub-workflow or job triggered by the workflow is associated with the same hog group as its parent.
+
+Thus:
+
+- Every workflow is assigned to exactly one hog group.
+- Every hog group may have many workflows assigned to it.
+
+The limits discussed below happen at the granularity of the hog group.
+
+### Hog Factor
+
+The hog factor is an integer greater than or equal to 1. It represents a trade-off between: 
+
+- Maximizing the resources of Cromwell and allowing jobs to be processed as fast as possible.
+- Remaining responsive to small requests from other groups - even if a few greedy groups have enough work that they 
+could otherwise max out Cromwell's limits by themselves.
+
+Here are a few mental models which might be helpful to thinking about the hog factor:
+
+- A hog factor of 2 means that "2 greedy users would be able to hog the entire resources of Cromwell" 
+- A hog factor of 100 means "any 1 group is only ever allowed to use 1/100th of the resources of the total Cromwell server"
+
+### Hog Limit
+
+Hog Limits are not set directly, but are values that Cromwell calculates internally. 
+They are the evaluated upper bounds on how much of a given resource type could be given, in total, to workflows within 
+a single greedy group (for example in the sentence "this Cromwell has calculated a hog limit of 2 jobs per group"). 
+
```

Comment:
Rephrase/reorder:
>A hog limit is how much of a given resource that a hog group could use. Hog limits are not set directly; they are values that Cromwell calculates internally. 
For example, a single hog group may be limited by Cromwell to a hog limit of 2 jobs per group. Therefore no matter how many workflows, sub-workflows, and jobs are in a greedy hog group, the whole group is limited to 2 concurrent jobs.